examples/react-native/babel.config.js

```diff
@@ -3,4 +3,4 @@ module.exports = function (api) {
   return {
     presets: ['babel-preset-expo'],
   };
-};
+};
```

Comment:
there are few linting issues, you probably can fix them all at once using the linter from the root of the repo:
```
yarn lint:fix
```

Reply:
should the build fail if there are linting issues?

Reply:
ah the examples directory isnt part of the root project

Reply:
got the linter to run off the same config as in the root, it says there are no issues in here